### PR TITLE
Try to fix the coordinate mess

### PR DESCRIPTION
**Pull request by abcdefg30**

### CODE REVIEW

File: src/blueblock/Locator.java [hunk 1]
```diff
@@ -51,55 +43,46 @@ public static Human GetHuman(int x, int y) {
...
-		if (X0 < 0)
-			X0 = 0;
-		int XG = h.GetX() + 1;
-		if (XG >= Main.FieldColumns)
-			XG = h.GetX();
+		int pos1 = h.GetX();
+		int pos2 = h.GetY();
 		backgroundColor(pos1, pos2);
+
+		// TODO: x and y are somehow swapped?
```
Comment (abc013) [inline src/blueblock/Locator.java]: If you have enough time, you could eventuelly fix this.
Else we merge it and i will try.

File: src/blueblock/Main.java [hunk 2]
```diff
@@ -239,122 +246,65 @@ public void mouseClicked(MouseEvent e) {
...
 				break;
 			case "5":
-				H4.Go("down");
+				H4.Go(confused ? "down" : "up");
 				break;
 			case "2":
-				H4.Go("up");
+				H4.Go(confused ? "up" : "down");
 				break;
-			}
```
Comment (abc013) [inline src/blueblock/Main.java:357]: 👍 Good Job.

**Review by abc013 — APPROVED** [review]
Code looks fine, just have to test this in the game 👍 .

**Comment by abc013** [comment]
- so, i just found out that the numpad (the keys for Player 4) don't work.
Can you correct that?
- just to mention it here, don't Forget to do #9 😆.

**Comment by abcdefg30** [comment]
I think I fixed everything now.

**Review by abc013 — APPROVED** [review]
Very good, haven't tested this, but it think this can get in.

**Pull request merged**